### PR TITLE
DS-558 Deprecate color indigo

### PR DESCRIPTION
**Pull request by MarcinMr**
## Jira

https://pegadigitalit.atlassian.net/browse/DS-558

## Summary

Any `indigo` usage was converted to `navy`.

## How to test

Make sure there is no more colour `indigo` present in the repo.

## Release notes

The `indigo` colour has been renamed to `navy`.

### CODE REVIEW

File: packages/core-v3.x/styles/01-settings/settings-colors/_settings-colors.scss [hunk 1]
```diff
@@ -13,62 +13,55 @@ $bolt-color-default: 'base';
...
 /// @type map
 $bolt-brand-colors: (
-  'indigo': (
-    'xdark': hsl(220.4, 71%, 12%),
-    'dark': hsl(220.4, 80%, 15%),
-    'base': hsl(220.4, 100%, 18.6%),
-    'light': hsl(220.4, 50%, 49%),
-    'xlight': hsl(220.4, 73%, 81%)
-  ),
   'navy': (
```
Comment (mikemai2awesome) [inline packages/core-v3.x/styles/01-settings/settings-colors/_settings-colors.scss:16]: We want to deprecate first, make sure Pattern Lab itself doesn't use indigo anymore. We cannot remove yet.

File: docs-site/src/pages/pattern-lab/_patterns/30-layouts/layout/-45-layout-item-vertical-alignment.twig [hunk 2]
```diff
@@ -95,7 +95,7 @@
         <bolt-layout template="halves@from-small 33/67@from-medium" padding-top="large" padding-bottom="large" gutter="large" valign-items="center" class="t-bolt-light">
           <bolt-layout-item valign-self="start-offset">
             {% include '@bolt-components-image/image.twig' with {
-              src: '/images/heros/hero-foreground--indigo.png',
+              src: '/images/heros/hero-foreground--navy.png',
```
Comment (mikemai2awesome) [inline docs-site/src/pages/pattern-lab/_patterns/30-layouts/layout/-45-layout-item-vertical-alignment.twig:98]: This is a file name, it should not be changed.

**Review by mikemai2awesome — CHANGES_REQUESTED** [review]
@MarcinMr We are deprecating not removing. Please roll back some changes.

File: docs-site/src/pages/pattern-lab/_patterns/10-visual-styles/00-color/00-color-system.twig [hunk 3]
```diff
@@ -18,14 +18,12 @@
...
         {% for color_name, palette in bolt.data.colors.brand %}
-          {% if color_name != "indigo" %}
-            {% include "_color-swatch.twig" with {
-              color_swatch: {
-                color_name: color_name,
-                palette: palette
-              }
-            } %}
-          {% endif %}
+          {% include "_color-swatch.twig" with {
```
Comment (mikemai2awesome) [inline docs-site/src/pages/pattern-lab/_patterns/10-visual-styles/00-color/00-color-system.twig:21]: Do not remove the condition here. It'll be done when we are removing indigo.

File: docs-site/src/pages/pattern-lab/_patterns/10-visual-styles/00-color/05-color-tokens.twig [hunk 4]
```diff
@@ -13,48 +13,46 @@
...
 
       {% for color_name, palette in bolt.data.colors.brand %}
-        {% if color_name != "indigo" %}
-          <bolt-text headline font-size="xlarge" tag="h3">
-            {{ color_name|capitalize }}
-          </bolt-text>
-          <bolt-table first-col-fixed-width>
-            <table>
-              <thead>
+        <bolt-text headline font-size="xlarge" tag="h3">
```
Comment (mikemai2awesome) [inline docs-site/src/pages/pattern-lab/_patterns/10-visual-styles/00-color/05-color-tokens.twig:16]: Same here, do not remove the condition.

File: docs-site/src/pages/pattern-lab/_patterns/10-visual-styles/00-color/05-color-tokens.twig [hunk 5]
```diff
@@ -13,48 +13,46 @@
...
+        </bolt-table>
       {% endfor %}
     </bolt-tab-panel>
 
     <bolt-tab-panel>
       <bolt-text slot="label">Status Colors</bolt-text>
 
       {% for color_name, palette in bolt.data.colors.status %}
-        {% if color_name != "indigo" %}
+        {% if color_name != "navy" %}
```
Comment (mikemai2awesome) [inline docs-site/src/pages/pattern-lab/_patterns/10-visual-styles/00-color/05-color-tokens.twig:55]: Revert this.

File: docs-site/src/pages/pattern-lab/_patterns/10-visual-styles/00-color/15-text-color-utilities.twig [hunk 6]
```diff
@@ -13,7 +13,6 @@
       <bolt-text slot="label">Brand Colors</bolt-text>
 
       {% for color_name, palette in bolt.data.colors.brand %}
-        {% if color_name != "indigo" %}
           <bolt-text headline font-size="xlarge" tag="h3">
```
Comment (mikemai2awesome) [inline docs-site/src/pages/pattern-lab/_patterns/10-visual-styles/00-color/15-text-color-utilities.twig:16]: Same here, do not remove the condition.

File: docs-site/src/pages/pattern-lab/_patterns/10-visual-styles/00-color/15-text-color-utilities.twig [hunk 7]
```diff
@@ -42,15 +41,14 @@
...
-        {% endif %}
       {% endfor %}
     </bolt-tab-panel>
 
     <bolt-tab-panel>
       <bolt-text slot="label">Status Colors</bolt-text>
 
       {% for color_name, palette in bolt.data.colors.status %}
-        {% if color_name != "indigo" %}
+        {% if color_name != "navy" %}
```
Comment (mikemai2awesome) [inline docs-site/src/pages/pattern-lab/_patterns/10-visual-styles/00-color/15-text-color-utilities.twig:51]: Revert his.

File: docs-site/src/pages/pattern-lab/_patterns/40-components/hero/50-hero--image-valign-top.twig [hunk 8]
```diff
@@ -32,8 +32,8 @@
       }
     }),
   ]|join(''),
-  background: "/images/heros/hero-background--indigo.jpg",
-  image: "/images/heros/hero-foreground--indigo.png",
+  background: "/images/heros/hero-background--navy.jpg",
```
Comment (mikemai2awesome) [inline docs-site/src/pages/pattern-lab/_patterns/40-components/hero/50-hero--image-valign-top.twig:35]: These are file names. Do not change.

File: docs-site/src/pages/pattern-lab/_patterns/40-components/icon/15-icon-color-variation.twig [hunk 9]
```diff
@@ -2,20 +2,18 @@
...
-              name: "marketing-gray",
-              size: "large",
-              color: color,
-              background: background,
-            } only %}
-          {% endfor %}
-        </div>
-      </bolt-list-item>
-    {% endif %}
+    <bolt-list-item>
```
Comment (mikemai2awesome) [inline docs-site/src/pages/pattern-lab/_patterns/40-components/icon/15-icon-color-variation.twig:5]: Same here, do not remove the condition.

**Review by mikemai2awesome — CHANGES_REQUESTED** [review]

File: packages/core-v3.x/styles/01-settings/settings-colors/_settings-colors.scss (hunk 1 above)
Comment (MarcinMr) [reply packages/core-v3.x/styles/01-settings/settings-colors/_settings-colors.scss:23]: Reverted

File: docs-site/src/pages/pattern-lab/_patterns/30-layouts/layout/-45-layout-item-vertical-alignment.twig (hunk 2 above)
Comment (MarcinMr) [reply docs-site/src/pages/pattern-lab/_patterns/30-layouts/layout/-45-layout-item-vertical-alignment.twig]: Reverted

File: docs-site/src/pages/pattern-lab/_patterns/10-visual-styles/00-color/00-color-system.twig (hunk 3 above)
Comment (MarcinMr) [reply docs-site/src/pages/pattern-lab/_patterns/10-visual-styles/00-color/00-color-system.twig]: Reverted

File: docs-site/src/pages/pattern-lab/_patterns/10-visual-styles/00-color/05-color-tokens.twig (hunk 4 above)
Comment (MarcinMr) [reply docs-site/src/pages/pattern-lab/_patterns/10-visual-styles/00-color/05-color-tokens.twig]: Reverted

File: docs-site/src/pages/pattern-lab/_patterns/10-visual-styles/00-color/05-color-tokens.twig (hunk 5 above)
Comment (MarcinMr) [reply docs-site/src/pages/pattern-lab/_patterns/10-visual-styles/00-color/05-color-tokens.twig]: Reverted

File: docs-site/src/pages/pattern-lab/_patterns/10-visual-styles/00-color/15-text-color-utilities.twig (hunk 7 above)
Comment (MarcinMr) [reply docs-site/src/pages/pattern-lab/_patterns/10-visual-styles/00-color/15-text-color-utilities.twig]: Reverted

File: docs-site/src/pages/pattern-lab/_patterns/40-components/hero/50-hero--image-valign-top.twig (hunk 8 above)
Comment (MarcinMr) [reply docs-site/src/pages/pattern-lab/_patterns/40-components/hero/50-hero--image-valign-top.twig]: Reverted

File: docs-site/src/pages/pattern-lab/_patterns/40-components/icon/15-icon-color-variation.twig (hunk 9 above)
Comment (MarcinMr) [reply docs-site/src/pages/pattern-lab/_patterns/40-components/icon/15-icon-color-variation.twig]: Reverted

**Comment by MarcinMr** [comment]
@mikemai2awesome change requests reverted

**Review by mikemai2awesome — APPROVED** [review]
@danielamorse This has some Academy changes, please review.

**Review by danielamorse — CHANGES_REQUESTED** [review]
I reviewed the Academy changes and made [a ticket](https://pegadigitalit.atlassian.net/browse/ACD-957) to update colors in Academy codebase.

@MarcinMr @mikemai2awesome be sure to include a "Release notes" section to your PR description when there are deprecations. See [this PR](https://github.com/boltdesignsystem/bolt/pull/2229) for reference 🙏

**Comment by MarcinMr** [comment]
@danielamorse I added release notes to this PR, let me know if that's ok.

**Review by danielamorse — APPROVED** [review]
Thanks for updating the "Release notes" @MarcinMr. I just removed the "Deprecations" heading.

We don't have an official format for this, but here's how the notes get compiled for [an actual release](https://github.com/boltdesignsystem/bolt/releases/tag/v4.2.0). Maybe in a future dev huddle we can decide on a good format and document it.

**Pull request merged**